libs/sdk-core/src/persist/db.rs

```diff
@@ -205,6 +205,20 @@ impl SqliteStorage {
             M::up("
              ALTER TABLE payments_external_info ADD COLUMN lnurl_metadata TEXT;
             "),
+            M::up("
+             CREATE TABLE IF NOT EXISTS reverse_swaps (
+               id TEXT PRIMARY KEY NOT NULL,
+               created_at INTEGER NOT NULL DEFAULT CURRENT_TIMESTAMP,
+               local_preimage BLOB NOT NULL UNIQUE,
+               local_private_key BLOB NOT NULL UNIQUE,
+               destination_address TEXT NOT NULL,
+               boltz_api_status TEXT NOT NULL,
```

Comment:
We need to split (different tables) the reverse swap immutable data from the data that can be retrieved using the API and chain information.
This way we can sync the immutable data only and every app can build the dynamic data itself.
Similar to the btc_swap feature